CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # CHANGELOG
 
 ## [next]
+ - Fix Box2DComponent render priority
```

Comment:
For this particular case should we have a ## 1.0 on top of `next`?

Reply:
If we merge into the 1.0 branch and rebase that one on master when we release 0.23.0 I think we should be fine?

Reply:
Fixed